guide/C/ch_loans.xml

```diff
@@ -379,7 +379,7 @@ Discrete Interest:   <emphasis>ieff = (1 + i/CF)^(CF/PF) - 1</emphasis>
 Continuous Interest: <emphasis>ieff = e^(i/PF) - 1 = exp(i/PF) - 1</emphasis>
 
 To convert from ieff to i, the following expressions are used:
-Discrete Interst:    <emphasis>i = CF*[(1+ieff)^(PF/CF) - 1]</emphasis>
+Discrete Interest:    <emphasis>i = CF*[(1+ieff)^(PF/CF) - 1]</emphasis>
```

Comment:
Perhaps we should 

1. put spaces around operators
2. replace `^(PF/CF)` by `<superscript>PF/CF</superscript>`
3. define i-eff as entity `i<<subscript>eff</subscript>`
4. Further improvements: equation, mathphrase